tests/test_plan.py

```diff
@@ -162,3 +168,55 @@ def test_mesh_cell_faces_with_output(tmp_path):
     valid = get_sha1_hash(TEST_JSON / "bald-eagle-mesh-cell-faces.geojson")
     test = get_sha1_hash(temp_faces)
     assert valid == test
+
+
+def test_mesh_timeseries_output():
+    with RasPlanHdf(BALD_EAGLE_P18_TIMESERIES) as plan_hdf:
+        with pytest.raises(ValueError):
+            plan_hdf.mesh_timeseries_output(
+                "Fake Mesh", TimeSeriesOutputVar.WATER_SURFACE
+            )
+        with pytest.raises(ValueError):
+            plan_hdf.mesh_timeseries_output("BaldEagleCr", "Fake Variable")
+
+
+def test_mesh_timeseries_output_cells():
+    with RasPlanHdf(BALD_EAGLE_P18_TIMESERIES) as plan_hdf:
+        ds = plan_hdf.mesh_timeseries_output_cells("BaldEagleCr")
+        assert "time" in ds.coords
```

Comment:
Would it make sense to be more specific and instead of just checking if time/cell_id is in the ds coords, we ensure that they equal the value that they should?

Reply:
Yeah, good point, I'll add some checks for that.